first_order_model/config/paper_configs/encoder_effect/lr128_no_encoder.yaml

```diff
@@ -1,6 +1,6 @@
 dataset_params:
-  root_dir: /video-conf/scratch/pantea/fom_personalized_1024
-  frame_shape: [1024, 1024, 3]
+  root_dir: /video-conf/scratch/pantea/fom_personalized_512
```

Comment:
Please do not change any of my existing files in paper_configs, please revert this and use a different file instead